parseRecipes.js

```diff
@@ -9,9 +9,19 @@ var difficulties = {
   'Beginner': 1
 };
 
-var categoryOrder = ['General Usage', 'Offline', 'Beyond Offline', 'Performance', 'Web Push'];
+var categories = module.exports.categories = [
+  { title: 'General Usage', slug: 'general-usage' },
```

Comment:
I don't think we need it to be honest -- those keywords are everywhere in each demo.